policies/before_resolution/registry.rego

```diff
@@ -0,0 +1,37 @@
+package otel
+
+# A registry `attribute_group` containing at least one `ref` attribute is
+# considered invalid if it's not in the registry group.
+deny[attr_registry_violation("registry_with_ref_attr", group.id, attr.ref)] {
+	group := input.groups[_]
+	startswith(group.id, "registry.")
```

Comment:
non-blocking, just some thoughts for the future:
 
I hope we will invent a more formal way to define a registry.
E.g. 
- now someone can add `registry.foo.bar` in any place in the repo. 
- or we'll add metrics registry and will need a different prefix.
- the attribute definition in the registry won't even have `ref` from the schema perspective
- etc